docs/configuration/configuration.md

```diff
@@ -1518,6 +1560,10 @@ serverset_sd_configs:
 triton_sd_configs:
   [ - <triton_sd_config> ... ]
 
+# List of Eureka service discovery configurations.
+eureka_sd_configs:
```

Comment:
can you order that between ec2 and file?

Reply:
ok